src/widgets/programs/programs.resource.tsx

```diff
@@ -13,7 +13,26 @@ export function fetchPatientPrograms(
   );
 }
 
-type PatientPrograms = {
+export function getPatientProgramByUuid(
+  programUuid: string
+): Observable<PatientProgram> {
+  return openmrsObservableFetch(
+    `/ws/rest/v1/programenrollment/${programUuid}`
+  ).pipe(map(({ data }) => mapToPatientProgram(data)));
+}
+
+function mapToPatientProgram(data: any): PatientProgram {
```

Comment:
you can get fancy and do this:
```suggestion
function mapToPatientProgram(data): PatientProgram {
      return {...data}
}
```
perhaps this shouldn't even be a function at all 😄